pkg/neorpc/result/peer_heights.go

```diff
@@ -0,0 +1,64 @@
+package result
+
+import (
+	"encoding/json"
+	"strconv"
+)
+
+type (
+	GetPeerHeights struct {
```

Comment:
Here and below, exported structure needs a comment of the form "// GetPeerHeights is a...".